hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -219,6 +220,21 @@ def ast_arguments_matches_signature(args, sig):
     return expected == [(p.name, p.kind) for p in sig.parameters.values()]
 
 
+def is_func_param_called_within(f, name):
+    """Is the given name referenced within f?"""
+    try:
+        tree = ast.parse(textwrap.dedent(inspect.getsource(f)))
+    except (OSError, ValueError):
+        return True
```

Comment:
The simplest way to trigger this for full coverage is probably to call it on a builtin, e.g. `is_func_param_called_within(object, "")`.

Reply:
We ended up calling it on a `compile`d string. Do we change it or leave it as-is?

Reply:
Compiled string is great too 👍